actions/index.js

```diff
@@ -26,4 +26,9 @@ export const setCurrentFly = (data) => ({
 export const setSelectedFlyId = (data) => ({
 	type: 'SET_FLY_ID',
 	data
+})
+
+export const setFish = data => ({
+	type: 'SET_FISH',
+	data
```

Comment:
Thank you for setting this up to store the fish data.  This will be useful when I use the fish data in the Fish Form component.